en/micro-integrator/docs/references/connectors/mongodb-connector/mongodb-connector-config.md

```diff
@@ -0,0 +1,1774 @@
+            Gssapi Service Name
+        </td>
+        <td>
+            String
+        </td>
+        <td>
+            Set the Kerberos service name when connecting to Kerberized MongoDB instances. This value must match the service name set on MongoDB instances to which you are connecting.
+        </td>
+        <td>
+            -
+        </td>
+        <td>
+            No
+        </td>
+    </tr>
+    <tr>
+        <td>
+            W
+        </td>
+        <td>
+            String
+        </td>
+        <td>
+            Corresponds to the write concern w Option. The w option requests acknowledgement that the write operation has propagated to a specified number of mongod instances or to mongod instances with specified tags.
+        </td>
+        <td>
+            -
+        </td>
+        <td>
+            No
+        </td>
+    </tr>
+    <tr>
+        <td>
+            W Timeout MS
+        </td>
+        <td>
+            Number
+        </td>
+        <td>
+            Corresponds to the write concern wtimeout. wtimeoutMS specifies a time limit, in milliseconds, for the write concern.
+        </td>
+        <td>
+            -
+        </td>
+        <td>
+            No
+        </td>
+    </tr>
+    <tr>
+        <td>
+            Journal
+        </td>
+        <td>
+            String
+        </td>
+        <td>
+            Corresponds to the write concern j Option option. The journal option requests acknowledgement from MongoDB that the write operation has been written to the journal. 
+        </td>
+        <td>
+            -
+        </td>
+        <td>
+            No
+        </td>
+    </tr>
+    <tr>
+        <td>
+            Maximum Pool Size
+        </td>
+        <td>
+            Number
+        </td>
+        <td>
+            The maximum number of connections in the connection pool.
+        </td>
+        <td>
+            100
+        </td>
+        <td>
+            No
+        </td>
+    </tr>
+    <tr>
+        <td>
+            Minimum Pool Size
+        </td>
+        <td>
+            Number
+        </td>
+        <td>
+            The minimum number of connections in the connection pool.
+        </td>
+        <td>
+            0
+        </td>
+        <td>
+            No
+        </td>
+    </tr>
+    <tr>
+        <td>
+            Maximum Idle Time MS
+        </td>
+        <td>
+            Number
+        </td>
+        <td>
+            The maximum number of milliseconds that a connection can remain idle in the pool before being removed and closed.
+        </td>
+        <td>
+            -
+        </td>
+        <td>
+            No
+        </td>
+    </tr>
+    <tr>
+        <td>
+            Wait Queue Multiple
+        </td>
+        <td>
+            Number
+        </td>
+        <td>
+            A number that the driver multiples the maxPoolSize value to, to provide the maximum number of threads allowed to wait for a connection to become available from the pool. 
```

Comment:
Not sure what this means.

Reply:
Resolved in the commit https://github.com/shiromi-basil/docs-ei/commit/3351838abc8a6a10ff1350ac9f9d580138e4956b.